tools/analyzer_plugin/lib/src/plugin.dart

```diff
@@ -94,9 +94,9 @@ class OverReactAnalyzerPlugin extends ServerPlugin
       ..performanceLog = performanceLog
       ..fileContentOverlay = fileContentOverlay;
     final result = contextBuilder.buildDriver(root);
-    runZoned(() {
+    runZonedGuarded(() {
       result.results.listen(processDiagnosticsForResult);
-    }, onError: (e, stackTrace) {
+    }, (e, stackTrace) {
```

Comment:
`onError` is `dynamic` so they deprecated it and said to use runZonedGuarded instead, which has a typed callback